pandas/tests/series/test_operators.py

```diff
@@ -976,16 +984,24 @@ def run_ops(ops, get_ser, test_ser):
         # ## timedelta64 ###
         td1 = Series([timedelta(minutes=5, seconds=3)] * 3)
         td1.iloc[2] = np.nan
-        td2 = timedelta(minutes=5, seconds=4)
-        ops = ['__mul__', '__floordiv__', '__pow__', '__rmul__',
-               '__rfloordiv__', '__rpow__']
-        run_ops(ops, td1, td2)
-        td1 + td2
-        td2 + td1
-        td1 - td2
-        td2 - td1
-        td1 / td2
-        td2 / td1
+        tdscalar = Timedelta(minutes=5, seconds=4)
+        ops = ['__mul__', '__pow__', '__rmul__', '__rpow__']
+        run_ops(ops, td1, tdscalar)
+        td1 + tdscalar
+        tdscalar + td1
+        td1 - tdscalar
+        tdscalar - td1
+        td1 / tdscalar
+        tdscalar / td1
+        tm.assert_series_equal(td1 // tdscalar, Series([0, 0, np.nan]))
```

Comment:
can you parametrize these last ones they seem pretty similar

Reply:
This is part of a 127 line test that is all over the place.  I'd prefer to clean it up in a follow-up.

Reply:
ok, this needs parameterization / refactor, but can follow up.